models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -183,6 +183,26 @@ public OpenAiApi(String baseUrl, String apiKey, MultiValueMap<String, String> he
 	 */
 	public enum ChatModel implements ChatModelDescription {
 
+		/**
```

Comment:
These models are a bit different compared to the rest. For example, they don't support function calling nor system messages. I would suggest considering how to convey this information to warn users that by switching model value they might get a broken application. It might be worth considering whether adding them to this enum is necessary. Even if they're not added to the enum, it's always possible to use these new models, either via configuration properties or via the ChatModel/ChatClient APIs.

Reply:
good point.  we need to think about it.  in anycase, i merged the enums in the interest of having folks explore.